types.d.ts

```diff
@@ -98,16 +98,6 @@ export interface SimulationData extends SimulationResult {
   config: SimulationConfig;
 }
 
-// --- Proposal checks ---
-export type ProposalActions = [
```

Comment:
is this no longer useful? for GovernorBravo? https://github.com/compound-finance/compound-protocol/blob/master/contracts/Governance/GovernorBravoDelegate.sol#L181-L192

while we're on the subject, the fact that all flavors of governor all seem to be sloshing around in the same files seems like a code smell

Reply:
It's unused in the entire codebase. Hence the reason for removing it